### ISSUE TITLE
nexus 5x android N can not open hotspot

### ISSUE THREAD

**Issue by foxundermoon** [Mygod/VPNHotspot#140]
# **after click  wifi hotspot**

![image](https://user-images.githubusercontent.com/2764106/71827111-7c9eb100-30da-11ea-9967-39531d4db3e7.png)

![image](https://user-images.githubusercontent.com/2764106/71826986-38abac00-30da-11ea-9dfe-3d78017b880c.png)

what's the problem detail?
on my Samsung s8 is ok.

**Comment by chevyfollowtooclose** [comment]
On Sprint I had to root Nexus 5x via instructions on xdadevelopers to get hotspot or tether to work and even still it would only work with this program. Note that many Verizon phones are locked so you can't ever root them. 

Your carrier pushes updates to your phone to block tethering (not legal in many countries that aren't the US) when you configure service for any phone.  This program is for that problem. 

Don't know why it is working on your Samsung but maybe you are paying for tethering thru carrier or the software is just different. It is a much newer phone. 

Anyway in summary your tethering is blocked by carrier and you would have to root in order to get this program to work.

**Comment by Mygod** [comment]
Known issue on Android 7. Update your system or just use system settings.
